### PR TITLE
Request to add credentials to members.yaml

### PR DESCRIPTION
**Pull request by amsaha**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/158)

### CODE REVIEW

**Comment by amsaha** [comment]
Background: https://github.com/kubeflow/kubeflow/issues/1202

File: members.yaml [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
-  firstName: Ankush
-  lastName: Agarwal
-  affiliation: Google
-  email: agwl@google.com
+- github: amsaha
+  slack: amsaha
+  firstName: Amit
+  lastName: Saha
+  affiliation: Cisco
+  email: amisaha@google.com
```
Comment (jlewi) [inline members.yaml]: Is this email address correct?

**Comment by jlewi** [comment]
Sent you an invite to the GitHub org.

File: members.yaml (hunk 1 above)
Comment (amsaha) [reply members.yaml]: Good catch :-) Let me fix it.

**Comment by amsaha** [comment]
Needs /ok-to-test label.

**Pull request merged**